packages/suite-web-landing/next-env.d.ts

```diff
@@ -1,2 +1,3 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+/// <reference path="../suite/global.d.ts" />
```

Comment:
i don't like this, looks like using "index exports" (Translation component in this case) is leading to importing unwanted parts of the code

Reply:
if we want to use `Translation` component between project it should be either placed in `@trezor/components` or imports shouldn't be written as `from "@suite/components"` but rather "@suite/components/AnotherComponent"

Reply:
It doesn't make sense to put this inside `@trezor/components` as it is not really an UI (meaning design) component. I would rather fix the issue (if it is an issue) another way.

I don't see any import from `"@suite/components"` (index file), only `@suite-types`. Is that what you meant?

Reply:
ah yes, `@suite-types` sorry.
looks like an issue to me, otherwise you will not need this change :)
suite-types are importing various files from the suite (actions, reducers etc.) and all those imported files are unnecessary bundled into landing page

Reply:
This was a nice exercise. I found out that reusing Translation component, without needing any import from suite that causes another bunch of imports, is pretty hard. I analysed if all these files are really the part of the bundle and I am not sure. The added first-load size with reused Translation component from suite is only 4kb (That is basically the size of one file).

My hunch is that the added types are needed because typescript needs to import basically everything to do a proper type-checking, but this doesn't mean that babel is gonna bundle all those things? Would be certainly interesting to do a more digging, maybe once things get more relaxed

<img width="657" alt="Screenshot 2020-10-20 at 13 08 47" src="https://user-images.githubusercontent.com/6961901/96579075-847cba00-12d6-11eb-90dd-97ee052c46f1.png">
<img width="676" alt="Screenshot 2020-10-20 at 13 11 41" src="https://user-images.githubusercontent.com/6961901/96578838-2223b980-12d6-11eb-9f39-7f96f6e9e125.png">